src/core/appgateway.tf

```
@@ -237,6 +237,23 @@ module "app_gw" {
       }
     }
 
+    api-web-io-pagopa-it = {
+      protocol           = "Https"
+      host               = format("api-web.%s.%s", var.dns_zone_io, var.external_domain)
+      port               = 443
+      ssl_profile_name   = format("%s-ssl-profile", local.project)
+      firewall_policy_id = azurerm_web_application_firewall_policy.api_app.id
```

```suggestion
      firewall_policy_id = azurerm_web_application_firewall_policy.api_app.id
```
Do you need a WAF policy in prevention mode? If no set to `null`

---

We want it. Do we need to create a new firewall policy, or is it ok?